species/fit/retrieval.py

```diff
@@ -1056,6 +1068,17 @@ def run_multinest(
         else:
             self.abund_nodes = None
 
+        # set default gradient priors if applicable
+        default_grad_priors = {'1':(0.25,0.025),'2':(0.25,0.045),'3':(0.26,0.05),
+                               '4':(0.2,0.05),'5':(0.12,0.045),'6':(0.07,0.07)}
+
+        if pt_profile == "gradient":
+            if prior is None:
+                prior = {}
+            for i in range(1,7):
+                if "PTslope_"+str(i) not in prior.keys():
```

Comment:
- I think it would also work without the `.keys()`
- The string could be change to an f-string: `f"PTslope_{i}"`